pkg/reader/ccip.go

```diff
@@ -57,6 +56,28 @@ func newCCIPChainReaderInternal(
 	destChain cciptypes.ChainSelector,
 	offrampAddress []byte,
 	addrCodec cciptypes.AddressCodec,
+) (*ccipChainReader, error) {
+	return newCCIPChainReaderWithConfigPollerInternal(
+		ctx,
+		lggr,
+		contractReaders,
+		contractWriters,
+		destChain,
+		offrampAddress,
+		addrCodec,
+		nil,
```

Comment:
nit: you can initialize it here and return an error if a nil config poller is passed into `newConfigPoller(lggr, reader, defaultRefreshPeriod)`
```suggestion
		newConfigPoller(lggr, reader, defaultRefreshPeriod),
```